File: Unidad 3/C#/P8_U3.cs
using System;

namespace P8_U3 // Note: actual namespace depends on the project name.
{
    internal class Mayor_Menor
    {
        static void Main(string[] args)
        {
            int N1, N2;
            Console.WriteLine("Ingresa el primer número: ");
            N1 = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine("Ingresa el segundo número: ");
            N2 = Convert.ToInt32(Console.ReadLine());

            if (N1 > N2)
                Console.WriteLine(N1 + " es mayor que " + N2);
            else
                Console.WriteLine(N1 + " es menor que " + N2);

        }
    }
}

File: Unidad 3/JAVA/P8_U3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.p8_u3;

import java.util.Scanner;

/**
 *
 * @author LaboratorioLTI2
 */
public class P8_U3 {

    public static void main(String[] args) {
        Scanner sc= new Scanner(System.in);
            int N1, N2;
            System.out.println("Ingresa el primer número: ");
            N1 = sc.nextInt();
            System.out.println("Ingresa el segundo número: ");
            N2 = sc.nextInt();

            if (N1 > N2)
                System.out.println(N1 + " es mayor que " + N2);
            else
                System.out.println(N1 + " es menor que " + N2);
    }
}
